packages/venia-concept/src/.storybook/config.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { configure, addDecorator } from '@storybook/react';
 import { Adapter } from '@magento/venia-drivers';
 import store from '../store';
-import '../index.css';
+import '../../../venia-ui/lib/index.css';
```

Comment:
We should use the alias/absolute path rather than relative.

```suggestion
import '@magento/venia-ui/lib/index.css';
```

Please update all imports of `venia-ui` in `venia-concept` to match this pattern.